Client/Coordinators/LibraryCoordinator.swift

```diff
@@ -15,7 +15,7 @@ class LibraryCoordinator: BaseCoordinator, LibraryPanelDelegate {
     private let profile: Profile
     private let tabManager: TabManager
     private let libraryViewController: LibraryViewController
-    weak var parentCoordinator: LibraryCoordinatorDelegate?
+    weak var parentCoordinator: (LibraryCoordinatorDelegate & LibraryPanelDelegate)?
```

Comment:
It seems strange that the parent coordinator is also a LibraryPanelDelegate when the LibraryCoordinator is already a LibraryPanelDelegate. I would prefer having separate delegates instead.